### ISSUE TITLE
SASL/Kerberos support

### ISSUE THREAD

**Issue by cstigler** [alexguan/node-zookeeper-client#49]
Will this library support connections to Zookeeper using SASL and Kerberos/NTLM? I can't find any info on this.

If that's not currently possible, is it likely to be added soon? Or would love to hear your thoughts on how feasible it would be for us to add that kind of functionality in a PR, and what the best strategy to do so might be.

Thank you for the library, and your time!

**Comment by alexguan** [comment]
No, it's not supported yet. I will take a look at the effort of adding that support. And if you have something in your mind, feel free to share it with me.

**Comment by cstigler** [comment]
Thank you for the speedy reply! As we move towards requiring that feature, we'll definitely look into what implementation would look like, and let you know how that goes.

**Comment by Roaders** [comment]
Hi @alexguan - did you ever manage to look into this? I now require kerberos authentication and am considering doing a PR to add this myself but don't really know where to start. Any pointers would be much appreciated.
Thanks
